src/components/UserRolesEditor.vue

```diff
@@ -0,0 +1,317 @@
+<template>
+  <q-dialog v-model="opened" @hide="close">
+    <q-card style="width: 100%;">
+
+      <!-- Title -->
+      <q-card-section>
+        <div class="text-h5">Edit Team Roles for user <b>{{user.username}}</b></div>
+      </q-card-section>
+
+      <q-card-section class="scroll">
+          <!-- Team selection -->
+
+            <q-select
+                v-model="teamModel"
+                :options="options"
+                label="Select team"
+                use-input
+                input-debounce="0"
+                @update:model-value="onTeamSelected"
+                @filter="teamSelectFilter"
+            />
+
+            <SlideTransition>
```

Comment:
Why you don't use `q-slide-transition` https://quasar.dev/vue-components/slide-transition ?

Reply:
I totally agree with you, but for some reason that I don't understand, it doesn't work. Maybe is related to the fact that the table is inside a dialog, or that maybe I'm playing around with `height/max-height` properties, can't tell, sorry.
If you have any advice or idea on how to make it work I would appreciate. :man_shrugging:

Reply:
No, i don't have any idea 🥲  
So it works now, but i think in the future we should figure it out

Reply:
Totally agree with you. For the record, I tested different combinations and also, tried using a different version of Quasar, none of it made it work :disappointed: 
In any case, I left a TODO note for our future selves :smile: